_posts/2018-06-05-random-vue-tips.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "Random Vue Tips"
+date: 2018-06-05 10:17:26
+tags:
+- Vue
+- Development
+- JavaScript
+categories: development
+twitter_text: "Random Vue Tips"
+authors:
+- Chidozie Oragwu
```

Comment:
Your name isn't visible on the page when I run this locally. Changing this to `authors: Chidozie Oragwu` fixes the issue.